mobile/assets/i18n/fr-FR.json

```
@@ -20,17 +20,17 @@
   "album_thumbnail_card_item": "1 élément",
   "album_thumbnail_card_items": "{} éléments",
   "album_thumbnail_card_shared": " · Partagé",
-  "album_thumbnail_owned": "Possédé",
+  "album_thumbnail_owned": "Propriétaire",
   "album_thumbnail_shared_by": "Partagé par {}",
-  "album_viewer_appbar_delete_confirm": "Are you sure you want to delete this album from your account?",
+  "album_viewer_appbar_delete_confirm": "Êtes-vous certains de vouloir effacer cet album de votre compte ?",
```

```suggestion
  "album_viewer_appbar_delete_confirm": "Êtes-vous certain de vouloir effacer cet album de votre compte ?",
```
Vous de politesse = `certain`
https://www.question-orthographe.fr/question/certain-ou-certains/

---

Hello,

yes you are right. Except for Téléverser, it's really never used in common French speeking ( too long word)

"Charger" in french or "uploader" in frenglish

---

IMO "charger" is ok but imprecise to describe an upload. What about "transférer" ?

---

Maybe better yes